msteams-platform/resources/schema/manifest-schema.md

```diff
@@ -695,5 +679,23 @@ Define the properties your app uses to post a user activity feed.
 ```
 
 ***
->
->
+
+## defaultInstallScope
+
+**Optional** - string
+
+Specifies whether the scope offers an experience in the context of an individual user alone (personal), a team, a groupchat, or meetings. Options are:
```

Comment:
Could you please change it to following? 

Specifies the install scope defined for this app by default. This will be the option displayed on the button when a user tries to add the app.  Options are: